--- origsrc/cairo-dock-plugins-2.1.3-3/tomboy/src/tomboy-dbus.c	2010-02-13 05:00:04.000000000 -0600
+++ src/cairo-dock-plugins-2.1.3-3/tomboy/src/tomboy-dbus.c	2010-02-17 02:11:28.901673900 -0600
@@ -28,7 +28,7 @@
 
 static DBusGProxy *dbus_proxy_tomboy = NULL;
 
-extern struct tm *localtime_r (time_t *timer, struct tm *tp);
+extern struct tm *localtime_r (const time_t *timer, struct tm *tp);
 
 
 
